File: test-artifacts/py/TestLogStatementInserter/L0001_output.py
def test():
    args_sets = [[[2, 7, 11, 15], 9], [[3, 2, 4], 6], [[3, 3], 6]]
    for idx, args_set in enumerate(args_sets):
        f_gold(*args_set)
"-----------------"
### twoSum
from typing import *
def f_gold(nums: List[int], target: int) -> List[int]:
    helper = {}
    myexactlog(helper)
    for i, v in enumerate(nums):
        myexactlog(0)
        num = target - v
        myexactlog(num)
        if num in helper:
            myexactlog(0)
            myexactlog([helper[num], i])
            return [helper[num], i]
        helper[v] = i
        myexactlog(helper)
"-----------------"
test()

File: test-artifacts/js/TestLogStatementInserter/L0001_output.js
function test() {
    let args_sets = [
        [
            [2, 7, 11, 15], 9
        ],
        [
            [3, 2, 4], 6
        ],
        [
            [3, 3], 6
        ]
    ];
    for (let idx = 0; idx < args_sets.length; idx++) {
        let args_set = args_sets[idx];
        f_gold(...args_set);
    }
}
"-----------------"

function f_gold(nums, target) {
    let helper = {};
    console.log(JSON.stringify(helper, null, 2));
    for (let i = 0; i < nums.length; i++) {
        console.log(JSON.stringify('for #0', null, 2));
        let v = nums[i];
        console.log(JSON.stringify(v, null, 2));
        let num = target - v;
        console.log(JSON.stringify(num, null, 2));
        if (helper[num] !== undefined) {
            console.log(JSON.stringify('if #0', null, 2));
            return [helper[num], i];
        }
        helper[v] = i;
        console.log(JSON.stringify(helper, null, 2));
    }
}
"-----------------"
test()